README.md

```diff
@@ -22,8 +22,11 @@ Make sure you've installed [translate-shell](https://github.com/soimort/translat
 Usage
 -----
 
+- `:'<,'>VTranslate en:fr`
```

Comment:
Can we name this `:TranslateReplace` ? 
So we have:
- `:Translate`
- `:TranslateReplace`
and later on: 
- `:TranslateVisual`


And put it in 2nd position, so it's clear the base cmd is `:Translate`

Reply:
✔️